docs/pages/_app.js

```diff
@@ -50,6 +50,7 @@ ponyfillGlobal.muiDocConfig = {
     if (newDeps['@mui/x-data-grid-pro']) {
       newDeps['@mui/material'] = versions['@mui/material'];
       newDeps['@mui/x-data-grid'] = versions['@mui/x-data-grid'];
+      newDeps.exceljs = 'latest';
```

Comment:
`versions['exceljs`]` did not work ? Like we do for `@mui/material`

Reply:
`versions.exceljs` is not defined at this stage. 

To do so, I added it to `csbGetVersions` the `exceljs: 'latest'` property